synthesizer/process/src/cost.rs

```diff
@@ -77,6 +77,29 @@ pub fn execution_cost<N: Network>(process: &Process<N>, execution: &Execution<N>
     Ok((total_cost, (storage_cost, finalize_cost)))
 }
 
+/// Returns the *minimum* cost in microcredits to publish the given execution (total cost, (storage cost, finalize cost)).
+pub fn execution_cost_deprecated<N: Network>(
```

Comment:
Rename `execution_cost_deprecated` to something like `execution_cost_v1`. 

It is not deprecated yet until the migration occurs. And it is more consistent with the naming of `MAPPING_BASE_COST_V1` and `MAPPING_BASE_COST_V2`